File: hw/villagomez/u2/hw18ZooJS/ec.edu.espe.zoo.model/Monkey.js
const Primate = require('./Primate');
const Cage = require('./Cage'); 

class Monkey extends Primate {

    constructor(id, description, dateOnborn, gender, numberOfBones, cage) {
        super(id, description, dateOnborn, gender, numberOfBones, cage);
    }

    feed(food) {
        console.log("Feeding a monkey");
    }
    
}

module.exports = Monkey;

File: hw/pantoja/u2/hw18ZooInPythonAndJavaScript/ZooInPython/ec_edu_espe_zoo_model/Monkey.py
import sys
import os

sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
from ec_edu_espe_zoo_model.Primate import Primate


class Monkey(Primate):
    def __init__(self, id, description, date_onborn, gender, number_of_bones, cage):
        super().__init__(id, description, date_onborn, gender, number_of_bones, cage)

    def feed(self, food):
        print("Feeding a monkey")

